### PR TITLE
Pass access_token to namespace getter

### PR DESCRIPTION
**Pull request by CannonLock**
- Fixes register buttons for non admin users

### CODE REVIEW

File: web_ui/frontend/components/layout/AuthenticatedContent.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const AuthenticatedContent = ({
     if (data && checkAuthentication) {
       return checkAuthentication(data);
     } else {
-      return data?.authenticated !== undefined;
+      return !!data?.authenticated;
```
Comment (CannonLock) [inline web_ui/frontend/components/layout/AuthenticatedContent.tsx:72]: Note this fixes another issue that I saw where the previous solution would consider you authenticated even if data.authenticated == false.

**Comment by bbockelm** [comment]
@CannonLock - could you please create a write-up of the underlying issue itself for posterity?

**Comment by CannonLock** [comment]
https://github.com/user-attachments/assets/11f621c0-22b6-4d53-8f2d-4e1ac17e65e4

**Comment by CannonLock** [comment]
> @CannonLock - could you please create a write-up of the underlying issue itself for posterity?

https://docs.google.com/document/d/1z6VgMSGXzO49keZenvYTS2xAXGkFsyHWZTjFONum2Ew/edit?usp=sharing

**Comment by h2zh** [comment]
could you take a look at the failed linter test

**Review by h2zh — APPROVED** [review]

**Pull request merged**